fulltime-developer.md

```diff
@@ -1,39 +1,37 @@
 # Mr. Henry is looking for a fulltime developer
 
-*At Mr. Henry, we sail against the current of the wild web development seas.*
+We are looking for a fulltime developer to join our team.
 
-Let aside the bad metaphors, we're working on our own way to build for the web. Instead of hopping on every new framework that pops up, we choose to take a step back and work on something stubborn & better.
-
-Based on new, modern, and future-proof web standards, we're trying to build websites that work, everywhere, for everyone. For every *Shiny New Thing™*, we want to raise a thing that *Just Works™*.
+## Who we are
 
-If this tickles your curiosity, we'd like to talk to you.
-If you think we're 100% in the wrong, we'd still like to talk to you.
+Mr. Henry is an agency with a small team of passionate experts.  
+We're a mix of developers and designers working on online and offline projects.  
 
-## Who we are
+We work on branding, design and web development.  
+For many projects we do the full package, from branding to print and web deliverables.
 
-Mr. Henry is an internet-bureau, graphic design firm, development lab, birthplace of concepts and ideas & creative cave... We love what we do! Mr. Henry is a small team, working on both little and big projects, be it on- or offline; but all with the same vigor!
+We work closely together across teams and disciplines.  
+Always trying to improve our work, and learn from each other.
 
-Our 'clientèle' ranges from rockstars to architects, photographers to cities. Some of our clients include KOMONO, Eastpak, M HKA, Hugo & Marie, OkO, Vedett, Hi-ReS!, August, Sagmeister & Walsh, Us by Night, Kunstenpunt, YUST, our moms, city of Ghent/Antwerp/Hasselt, The Mill and [Mr. Henry themself](https://www.mrhenry.be).
+We're based in Antwerp, Belgium.
 
-Some links for some behind-the-scene action: [www](https://www.mrhenry.be) - [insta](https://instagram.com/wearemrhenry)  - [fb](https://fb.com/wearemrhenry) 
+[www](https://www.mrhenry.be) - [insta](https://instagram.com/wearemrhenry)
```

Comment:
Do we still want to include social profiles here?

FB is definitely a faux pass, but maybe insta is aswel?
Maybe just the website is sufficient here?

Reply:
Preference for Instagram to stay, since it shows a succinct and relevant overview of work.

Reply:
just our website is fine. also you need to have an insta account to see stuff which is a big no-no imho.

Reply:
We definitely need to reword `insta` to `instagram`.
`insta` is just too cringe :)

----

My main motivation to remove these social profiles, aside from their possible obsolescence, is that they are not representative of our development work.

But I can understand that they show a glimpse of Mr. Henry in general, which might be interesting to candidates.